sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/EpochCoordinator.scala

```diff
@@ -133,6 +133,13 @@ private[continuous] class EpochCoordinator(
 
   private var currentDriverEpoch = startEpoch
 
+  val INSTRUCTION_FOR_FEWER_CORES =
```

Comment:
Since this message should only be used in `checkTotalCores`, let's move it there.

Reply:
Moved.